src/app/catalog/components/issuer-catalog/issuer-catalog.component.ts

```diff
@@ -58,7 +52,15 @@ export class IssuerCatalogComponent extends BaseRoutableComponent implements OnI
 			value: 'andere',
 		},
 	];
-
+	labelDateAsc = this.translate.instant('General.dateAscending');
+	labelDateDesc = this.translate.instant('General.dateDescending');
+	sortOptions = [
```

Comment:
This is a lot of duplicate code, that I'm sure could be extracted to a common helper or something similar